qcm2290.c

```diff
@@ -0,0 +1,226 @@
+	{ "gcc_qupv3_wrap0_s2_clk", &gcc, 0x6c },
+	{ "gcc_qupv3_wrap0_s3_clk", &gcc, 0x6d },
+	{ "gcc_qupv3_wrap0_s4_clk", &gcc, 0x6e },
+	{ "gcc_qupv3_wrap0_s5_clk", &gcc, 0x6f },
+	{ "gcc_qupv3_wrap_0_m_ahb_clk", &gcc, 0x66 },
+	{ "gcc_qupv3_wrap_0_s_ahb_clk", &gcc, 0x67 },
+	{ "gcc_sdcc1_ahb_clk", &gcc, 0xef },
+	{ "gcc_sdcc1_apps_clk", &gcc, 0xee },
+	{ "gcc_sdcc1_ice_core_clk", &gcc, 0xf0 },
+	{ "gcc_sdcc2_ahb_clk", &gcc, 0x65 },
+	{ "gcc_sdcc2_apps_clk", &gcc, 0x64 },
+	{ "gcc_sys_noc_cpuss_ahb_clk", &gcc, 0x9 },
+	{ "gcc_sys_noc_usb3_prim_axi_clk", &gcc, 0x18 },
+	{ "gcc_usb30_prim_master_clk", &gcc, 0x5b },
+	{ "gcc_usb30_prim_mock_utmi_clk", &gcc, 0x5d },
+	{ "gcc_usb30_prim_sleep_clk", &gcc, 0x5c },
+	{ "gcc_usb3_prim_phy_com_aux_clk", &gcc, 0x5e },
+	{ "gcc_usb3_prim_phy_pipe_clk", &gcc, 0x5f },
+	{ "gcc_vcodec0_axi_clk", &gcc, 0x12c },
+	{ "gcc_venus_ahb_clk", &gcc, 0x12d },
+	{ "gcc_venus_ctl_axi_clk", &gcc, 0x12b },
+	{ "gcc_video_ahb_clk", &gcc, 0x35 },
+	{ "gcc_video_axi0_clk", &gcc, 0x3b },
+	{ "gcc_video_throttle_core_clk", &gcc, 0x49 },
+	{ "gcc_video_vcodec0_sys_clk", &gcc, 0x129 },
+	{ "gcc_video_venus_ctl_clk", &gcc, 0x127 },
+	{ "gcc_video_xo_clk", &gcc, 0x3d },
+	// { "gpu_cc_debug_mux", &gcc, 0xe3 },
+	// { "mc_cc_debug_mux", &gcc, 0x9b },
+	{ "measure_only_cnoc_clk", &gcc, 0x19 },
+	{ "measure_only_ipa_2x_clk", &gcc, 0xc2 },
+	{ "measure_only_snoc_clk", &gcc, 0x7 },
+	{ "measure_only_qpic_clk", &gcc, 0x9c },
+	{ "measure_only_qpic_ahb_clk", &gcc, 0x9e },
+	{ "measure_only_hwkm_km_core_clk", &gcc, 0xa0 },
+	{ "measure_only_hwkm_ahb_clk", &gcc, 0xa2 },
+	{ "measure_only_pka_core_clk", &gcc, 0xa3 },
+	{ "measure_only_pka_ahb_clk", &gcc, 0xa4 },
+	{ "measure_only_cpuss_gnoc_clk", &gcc, 0xa6 },
+
+	{ "disp_cc_mdss_ahb_clk", &gcc, 0x41, &disp_cc, 0x1a },
+	{ "disp_cc_mdss_byte0_clk", &gcc, 0x41, &disp_cc, 0x11 },
+	{ "disp_cc_mdss_byte0_intf_clk", &gcc, 0x41, &disp_cc, 0x12 },
+	{ "disp_cc_mdss_esc0_clk", &gcc, 0x41, &disp_cc, 0x13 },
+	{ "disp_cc_mdss_mdp_clk", &gcc, 0x41, &disp_cc, 0xe },
+	{ "disp_cc_mdss_mdp_lut_clk", &gcc, 0x41, &disp_cc, 0xf },
+	{ "disp_cc_mdss_non_gdsc_ahb_clk", &gcc, 0x41, &disp_cc, 0x1b },
+	{ "disp_cc_mdss_pclk0_clk", &gcc, 0x41, &disp_cc, 0xd },
+	{ "disp_cc_mdss_vsync_clk", &gcc, 0x41, &disp_cc, 0x10 },
+	{ "disp_cc_sleep_clk", &gcc, 0x41, &disp_cc, 0x24 },
+	{ "disp_cc_xo_clk", &gcc, 0x41, &disp_cc, 0x23 },
+
+	{ "gpu_cc_ahb_clk", &gcc, 0xe3, &gpu_cc, 0x10 },
+	{ "gpu_cc_crc_ahb_clk", &gcc, 0xe3, &gpu_cc, 0x11 },
+	{ "gpu_cc_cx_gfx3d_clk", &gcc, 0xe3, &gpu_cc, 0x1a },
+	{ "gpu_cc_cx_gmu_clk", &gcc, 0xe3, &gpu_cc, 0x18 },
+	{ "gpu_cc_cx_snoc_dvm_clk", &gcc, 0xe3, &gpu_cc, 0x15 },
+	{ "gpu_cc_cxo_aon_clk", &gcc, 0xe3, &gpu_cc, 0xa },
+	{ "gpu_cc_cxo_clk", &gcc, 0xe3, &gpu_cc, 0x19 },
+	{ "gpu_cc_gx_cxo_clk", &gcc, 0xe3, &gpu_cc, 0xe },
+	{ "gpu_cc_gx_gfx3d_clk", &gcc, 0xe3, &gpu_cc, 0xb },
+	{ "gpu_cc_sleep_clk", &gcc, 0xe3, &gpu_cc, 0x16 },
+
+	{ "mccc_clk", &gcc, 0x9b, &mc_cc, 0x220 },
```

Comment:
Here you do have a valid GCC mux value to write... https://github.com/andersson/debugcc/pull/18#discussion_r1158282902

Reply:
in short, it's not exactly necessary

Reply:
But the value is written to a register... should we skip that write entirely or does it simply not blow up on a bogus value?

Reply:
The value is masked out by the mux masks. `0x9b` and `0x220` look correct. So this part looks good. 

@andersson could you please merge this?

Reply:
The values do look correct (in the same ballpark as the others), I'm just ~complaining~ reminding about https://github.com/andersson/debugcc/pull/18#discussion_r1158282902 where it was decided to write a value of `0xfeedbeef & 0x1fff` to GCC's `mux_reg` at `0x62024`.

Reply:
hm yeah that seems a bit excessive

Reply:
I have pushed [#24](https://github.com/andersson/debugcc/pull/24) , it should
make MCCC declarations reflect reality by not requiring dummy &gcc or
NULL entries.